install_config/aggregate_logging.adoc

```diff
@@ -3299,3 +3299,107 @@ link:https://github.com/Scribery/tlog[Tlog].
 |Scrubbed invalid output characters as bytes.
 |===
 endif::openshift-origin[]
+
+== Manual Elasticsearch rollouts
+
+The latest release of the Aggregated Logging stack has updated the Elasticsearch
+Deployment Config object so that it no longer has a Config Change Trigger meaning
+any changes to the DC will not result in an automatic rollout. This was to prevent
+unintended restarts happening in the ES cluster which could create lots of shard
+balancing as cluster members restart.
+
```

Comment:
Can we add another paragraph to this intro along the lines of ...

----

We present two restart procedures, `rolling-restart`, and `full-restart`, where a rolling restart gracefully applies appropriate changes to the Elasticsearch cluster without down time (provided 3 masters are configured), and a full restart safely applies major changes without risk to existing data.

Reply:
Is it required that there are three masters for no downtime, or just more than one would allow this?